### DISCUSSION TITLE
4 Semester 8 Woche

### DISCUSSION THREAD

**Discussion by davidlzt** [MelonigeMelone/xCalendar#34] [Blogkanal]
# Protokoll Review-Meeting Team XCalendar
Diese Woche haben wir uns mit Review Methodiken beschäftigt. Dabei haben wir uns einen Teil unseres Codes angeschaut und analysiert. Das Ergebnis dieses Meetings ist dieses [Protokoll Review-Meeting Team XCalendar](https://github.com/MelonigeMelone/xCalendar/blob/e85c9c58d7af8501536495d9d0d2e521f8518339/Dokumente/ProtokollReviewMeeting.md).

Wir freuen uns bereits euch nächste Woche wieder zu sehen.

**Comment by Christian-2003** [comment]
Hallo zusammen,
das erscheint mir ein erfolgreiches Review-Meeting gewesen zu sein. In eurem Protokoll bemerkt ihr in der Schlussfolgerung, das ihr mehr Testfälle implementieren wollt. Das erscheint mir sinnvoll - Tests kann man in der Regel nicht genug haben.
Habt ihr denn sonst noch irgendwas beim Review hinsichtlich der untersuchten Komponente festgestellt, was in der Zukunft verbessert werden kann, oder sind die Tests die einzige Erkenntnis? Sollte nichts weiteres aufgefallen sein, dann ziehe ich meinen Hut vor euch. Unter Berücksichtigung der im Protokoll von euch genannten Kriterien wäre das nämlich eine beachtliche Leistung eurerseits.

Dann freue ich mich schon auf den nächsten Blog.
